indicators/tests/test_rf_views.py

```diff
@@ -38,6 +38,15 @@ def test_template_saved(self):
         # Note that the extra white space on either side of the tier should be trimmed before saving.
         self.assertEqual(LevelTierTemplate.objects.get(program=self.program).names, ['this', 'tha t'])
 
+    def test_illegal_chars(self):
+        self.client.force_login(self.tola_user.user)
+        response = self.client.post(
+            reverse('save_custom_template'),
+            {'program_id': self.program.id, 'tiers': ['this, ', ' that']},
```

Comment:
Did you want to test if theres colons as well or just commas is fine for an illegal character?

Reply:
Well that would make sense since that's the point of the ticket.